build/ts/app.ts

```diff
@@ -146,6 +146,15 @@ class Application {
       .on(State.updated, () => { this.parseContent(); });
   }
 
+  public showTeam(name: string) {
+    this.gracefulOut($(`.card:has(.sticker-team):not(:has(.sticker-team-${name}))`));
```

Comment:
This may be a little unclear for someone. The following could work in a similar way.

```js
const $cars = $('.card');
const $nonTeamCards = $cards
  .filter('.sticker-team')
  .filter(`:not(.sticker-team-${name})`);

gracefulOut($nonTeamCards);
```

Reply:
:+1: